dask/array/tests/test_array_core.py

```diff
@@ -617,6 +617,22 @@ def test_store():
     assert raises(ValueError, lambda: store([at, bt], [at, bt]))
 
 
+def test_to_hdf5():
+    try:
+        import h5py
+    except ImportError:
+        return
```

Comment:
That adds the unittest2 dependency which you only want to install if you're
in py26 and not otherwise.

 What is wrong with returning?

On Fri, May 8, 2015 at 4:59 PM, Stephan Hoyer <notifications@github.com>
wrote:

> In dask/array/tests/test_array_core.py
> <https://github.com/ContinuumIO/dask/pull/191#discussion_r29986020>:
>
> > @@ -617,6 +617,22 @@ def test_store():
> >      assert raises(ValueError, lambda: store([at, bt], [at, bt]))
> >
> >
> > +def test_to_hdf5():
> > +    try:
> > +        import h5py
> > +    except ImportError:
> > +        return
>
> The simple solution is to use import unittest2 as unittest when running
> Python 2.6.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ContinuumIO/dask/pull/191/files#r29986020>.
>